docker/docker.sh

```diff
@@ -6,6 +6,7 @@ set -e
 # use DEBUG=echo ./docker.sh to print all commands
 export DEBUG=${DEBUG:-""}
 export PUBLIC_PATH=${PUBLIC_PATH:-""}
+# export UNIT_PREF=unit_pref # TODO: Add this line to add this required environment variable? 
```

Comment:
Let's skip loading of units pref from .env. Just use metric units as default in the code.

Reply:
I have added `dotenv` in the `webpack.common.js` to let it read env variables from `.env` file too. Now everything works as expected.